### PR TITLE
nodejs/week3

### PR DESCRIPTION
**Pull request by Jul-S**

### CODE REVIEW

**Review by Saidemm — COMMENTED** [review]
Great job 💪

You are missing get requests for subtraction and division, and post for addition and subtraction.
You could try to optimize your code a little bit? Some parts of the code are repeating; for example: if (isNaN(Number(req.query[param])))
            return res.status(406).json({ error: "Not a number" }); 
by making a function and call it in all of your requests. 
 fx like this: function sendResponse(result, response) {
    isNaN(result) ? response.status(400).send("Input must be number") : response.send(`${result}`);
}

File: nodejs/week3/warmup/index.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
+const express = require("express");
+const app = express();
+
+app.use(express.urlencoded({ extended: true }));
+
+app.get("/", (req, res) => res.send("nodejs week3 homework"));
+
+app.get("/calculator/multiply", (req, res) => {
+    let result = 1;
```
Comment (Saidemm) [inline nodejs/week3/warmup/index.js]: Please consider using try and catch and send the Internal server error :
response.status(500).send({ error: "Internal Server Error." });

**Comment by Jul-S** [comment]
HI @Saidemm ! 
Thank you for feed back! Great idea with function!
Also Substraction and divisin is available through the request body using POST only, as it said in homework instructions.

**Comment by Jul-S** [comment]
I`ve updated myPR

**Pull request merged**